### PR TITLE
Fixes #8312: Rename poorly named tool \"signature.sh\"

### PR DESCRIPTION
**Pull request by jooooooon**
https://www.rudder-project.org/redmine/issues/8312

### CODE REVIEW

File: rudder-agent/SOURCES/signature.sh [hunk 1]
```diff
@@ -1,54 +1,9 @@
...
-keydate=${KEYDATE}
-keyid=${KEYID}
-EOF
+echo "WARNING: This command is deprecated and will be removed in a future version of Rudder."
+echo "WARNING: Use $(dirname $0)/rudder-sign instead."
+echo ""
+echo "Redirecting to $(dirname $0)/rudder-sign..."
+echo ""
 
+$(dirname $0)/rudder-sign $*
```
Comment (peckpeck) [inline rudder-agent/SOURCES/signature.sh]: `exec "$(dirname $0)/rudder-sign" "$@"`

Comment (peckpeck) [inline rudder-agent/SOURCES/signature.sh]: This script is called for the inventory technique, you should change the name there too.

**Comment by jooooooon** [comment]
Commit modified

**Comment by jooooooon** [comment]
Yes, that is https://www.rudder-project.org/redmine/issues/8313

**Comment by peckpeck** [comment]
OK, merging

**Pull request merged**